client/src/modules/CourseManagement/components/CourseTaskModal/index.tsx

```diff
@@ -38,8 +39,21 @@ export function CourseTaskModal(props: Props) {
     return userService.searchUser(searchText);
   };
 
+  const isInvalidCrossCheckDuration = (startDate: string, endDate: string) => {
+    const MIN_CROSSCHECK_DURATION = 3 * 24;
```

Comment:
Let's create constants for 3 and 24 so that it would be clear to everyone what kind of magic it is.

Reply:
@Pulya10c  look at the changes